src/state/ward/actions.ts

```diff
@@ -1,16 +1,32 @@
 import { isEmpty } from "lodash";
 import { Dispatch } from "redux";
-import { DiseaseDTO } from "../../generated";
+import { DiseaseDTO, WardDTO } from "../../generated";
```

Comment:
```suggestion
import { WardDTO } from "../../generated";
```

This is unused and can be safely removed